packages/storybook/stories/link.mdx

```diff
@@ -0,0 +1,44 @@
+import { RouteLink } from '@equisoft/design-elements-react';
+import { ArgTypes, Canvas, Meta } from '@storybook/blocks';
+import * as RouteLinkStories from './link.stories';
+
+<Meta of={RouteLinkStories} />
+
+# Link
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+## Definition
+Links are a combination of a web address (URL) and a clickable element (often a word or phrase, sometimes an image), that brings the user somewhere and that doesn’t affect the back or front-end [1][2]. 
+<Canvas of={RouteLinkStories.Default} />
+
+## Usage
+### When to use
+- Use when you have to bring the user somewhere else into or outside the application.
+- AND use when it won’t affect the application back and front-end [3].
+
+### When not to use
+- Do not use if the action of clicking the link doesn’t bring the user somewhere else.
+- Do not use when the action of clicking on the link will change the application state (either front or back end) [2].
```

Comment:
est-ce que tu peux expliquer ce point?

Reply:
Quand ça change le front-end ou back end c'est plus un bouton (action) qui trigger quelque chose, tandis que le lien fait juste naviguer.

Reply:
ok en gros ce point est compris dans le 1er point c-à-d., si le lien ne change pas l'application state, il ne "_bring pas the user somewhere else_". J'enlèverais le point tout simplement.